koocook_core/controllers/mixins.py

```diff
@@ -30,5 +33,6 @@ def comment(self, item_id: int) -> ControllerResponse:
         comment_fields.update({self.item_reviewed_field: self.model.objects.get(pk=item_id)})
         comment = self.comment_model(**comment_fields)
         comment.save()
+        LOGGER.info(f"{self.author.name} has commented [{comment.id}] on {self.item_reviewed_field}#{item_id}")
```

Comment:
The log for this seems a bit weird. Example `testuser has commented [2] on reviewed_recipe#1`